docs/source/modin/supported/index_supported.rst

```diff
@@ -54,9 +54,9 @@ Attributes
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``nlevels``                 | P                               | Only single Index supported.                       |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``array``                   | N                               |                                                    |
+| ``array``                   | Y                               |                                                    |
```

Comment:
how about hasnans?

Reply:
ok, seems actually not supported, but i believe it shows up in our API doc

Reply:
@sfc-gh-vbudati can you take care of this also

Reply:
done! I removed some sections because none of the APIs in that section are supported.